--- a/node_modules/react-native-alarmageddon/android/src/main/java/com/rnalarmmodule/AlarmReceiver.kt
+++ b/node_modules/react-native-alarmageddon/android/src/main/java/com/rnalarmmodule/AlarmReceiver.kt
@@ -315,6 +315,8 @@
             putExtra("id", id)
             putExtra("title", title)
             putExtra("body", body)
+            putExtra("snoozeEnabled", true)
+            putExtra("snoozeInterval", minutes)
         }
 
         val pendingIntent = PendingIntent.getBroadcast(
